features/step_definitions/support/hooks.js

```
@@ -148,12 +164,44 @@ Before(
+        //   args: [
+        //     "--headless",
+        //     `--user-data-dir=${userDataDir}`
+        //   ] 
+        // }); // Enable headless mode
+        // this.seleniumDriver = new Builder().withCapabilities(capabilities).build();
+
         // selenium driver setup
-        const capabilities = Capabilities.chrome();
-        capabilities.set('chromeOptions', { "w3c": false, args: ["--headless"] }); // Enable headless mode
-        const driver = new Builder().withCapabilities(capabilities).build();
+        let options = new ChromeSeleniumWebdriver.Options();
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'options' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)